qadence_protocols/measurements/robust_shadow.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import annotations
+
+from qadence import QuantumModel
+from qadence.blocks.abstract import AbstractBlock
+from torch import Tensor
+
+from qadence_protocols.measurements.utils_shadow import estimations
+
+
+def compute_measurements(
```

Comment:
I'm not too sure about the difference between these two functions.

Reply:
Ah yeah OK. Would it make sense (in another PR) to merge them and produce the right outcome depending on a "measurement" or "expectation" input ?

Reply:
I can do in another PR.